src/mvesuvio/oop/NeutronComptonProfile.py

```diff
@@ -0,0 +1,48 @@
+
+
+
+class NeutronComptonProfile:
```

Comment:
How about implementing this as

```python
from dataclasses import dataclass

@dataclass(frozen=True)
class NeutronComptonProfile:
    mass: float
    label: str
    intensity: float
    width: float
    center: float
    intensity_bounds: tuple[float, float]
    width_bounds: tuple[float, float]
    center_bounds: tuple[float, float]

```

(The type annotations are a blind guess as I'm not familiar with the rest of the code...)